modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/dashboard-whats-new/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```

Comment:
If this is not necessary, please remove it.

Reply:
Done.